apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/charSequenceExpectations.kt

```diff
@@ -205,7 +205,7 @@ fun <T : CharSequence> Expect<T>.endsWith(expected: CharSequence): Expect<T> =
  *
  * @since 0.9.0
  *
- * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.CharSequenceAssertionsSpec.endsWithChar
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.CharSequenceExpectationsSpec.endsWithChar
```

Comment:
Looks like those were named wrong in the first place. Can you rename them to ...Samples instead of ...Spec

Reply:
Done in this PR.